.storybook/main.js

```diff
@@ -3,4 +3,20 @@ module.exports = {
   features: {
     postcss: false,
   },
+  core: {
+    builder: 'webpack5',
+  },
+  staticDirs: ['../static'],
+  webpackFinal: async (config, { configType }) => {
+    // eslint-disable-next-line no-param-reassign
+    config.resolve = {
+      ...config.resolve,
+      fallback: {
+        ...config.resolve.fallback,
+        os: false,
```

Comment:
Can you explain this a bit more, I don't know this part of webpack well. Is this cause of the issue?